liquibase-standard/src/main/java/liquibase/statement/core/DatabaseTableIdentifier.java

```diff
@@ -0,0 +1,39 @@
+package liquibase.statement.core;
+public class DatabaseTableIdentifier{
```

Comment:
Since we are using Lombok in some other classes, I think we can also applied to this class. I think adding:
```
- @Getter
- @Setter
- @AllArgsConstructor
```
would be fine.